.github/workflows/create-design-review-issue.yml

```diff
@@ -2,8 +2,8 @@ name: Create Design Review Issue
 
 on:
   schedule:
-    # Every Wednesday at 00:00:00
-    - cron: '0 0 * * 3'
+    # At minute 1 (every hour).
+    - cron: '1 * * * *'
```

Comment:
We don't need to run this every hour, but the cadence will affect how long is the delay to close/pin from the most recently ended  session.